packages/desktop/views/dashboard/send-flow/views/SelectTokenView.svelte

```diff
@@ -58,9 +62,9 @@
     function onContinueClick(): void {
         setNewTransactionDetails({
             type: NewTransactionType.TokenTransfer,
-            assetId: selectedAssetId,
+            assetId: selectedAsset.id,
             rawAmount: undefined,
-            unit: undefined,
+            unit: getUnitFromTokenMetadata(selectedAsset?.metadata),
```

Comment:
not sure why we get the unit on this view vs the view for the inputting the amount (where in te future you could potentially change the token) the unit is attached to the amount